swift-5.5/Sources/index.swift

```diff
@@ -29,14 +29,15 @@ func main(req: RequestValue, res: RequestResponse) -> RequestResponse {
   let teams = Teams(client)
   let users = Users(client)
 
-  if(req.env?["APPWRITE_FUNCTION_ENDPOINT"] == nil || req.env?["APPWRITE_FUNCTION_API_KEY"] == nil) {
-      print("Environment variables are not set. Function cannot use Appwrite SDK.")
-  } else {
-    client
-      .setEndpoint(req.env?["APPWRITE_FUNCTION_ENDPOINT"]!)
-      .setProject(req.env?["APPWRITE_FUNCTION_PROJECT_ID"]!)
-      .setKey(req.env?["APPWRITE_FUNCTION_API_KEY"]!) 
-  }
+  guard let endpoint = req.env["APPWRITE_FUNCTION_ENDPOINT"],
+    let project = req.env["APPWRITE_FUNCTION_PROJECT_ID"],
+    let apiKey = req.env["APPWRITE_FUNCTION_API_KEY"] else {
+      return res.json(data: ["error": "Missing environment variables"])
```

Comment:
We need to log warning into console, but continue the code if vars are nor provided. 

If they are provided, we need to set them on client, so person can use Appwrite.

Can you please double-check this logic? As far as I am concern, only this res.json needs to be updated with print, but I am not experienced with Swift